tests/common/integration_features/test_fixes_integration.py

```diff
@@ -26,6 +38,102 @@ def test_integration_valid(self):
+            'CKV_AWS_130': {
+                'id': 'BC_AWS_NETWORKING_53',
+                'title': 'Ensure VPC subnets do not assign public IP by default',
+                'guideline': 'https://docs.bridgecrew.io/docs/ensure-vpc-subnets-do-not-assign-public-ip-by-default',
+                'severity': 'MEDIUM',
+                'pcSeverity': 'MEDIUM',
+                'category': 'Networking',
+                'checkovId': 'CKV_AWS_130',
+                'constructiveTitle': 'Ensure VPC subnets do not assign public IP by default',
+                'descriptiveTitle': 'AWS VPC subnets should not allow automatic public IP assignment',
+                'pcPolicyId': '11743cd3-35e4-4639-91e1-bc87b52d4cf5',
+                'additionalPcPolicyIds': ['11743cd3-35e4-4639-91e1-bc87b52d4cf5'],
+                'benchmarks': {}
+            }
+        }
+
+        metadata_integration.bc_to_ckv_id_mapping = {
+            'BC_AWS_NETWORKING_53': 'CKV_AWS_130'
+        }
+
+        report = Report(CheckType.TERRAFORM)
+
+        file = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'resources', 'main.tf')
+
+        report.add_record(Record(
+            check_id='CKV_AWS_130',
+            bc_check_id='BC_AWS_NETWORKING_53',
+            check_result={'result': CheckResult.FAILED, 'evaluated_keys': ['map_public_ip_on_launch']},
+            check_name='Ensure VPC subnets do not assign public IP by default',
+            check_class='checkov.terraform.checks.resource.aws.SubnetPublicIP',
+            code_block=[(1, 'resource "aws_subnet" "s" {\n'), (2, '  map_public_ip_on_launch = true\n'), (3, '}\n')],
+            evaluations=None,
+            file_abs_path=file,
+            file_line_range=[2, 3],
+            file_path='/main.tf',
+            resource='aws_subnet.s'
+        ))
+
+        report.add_record(Record(
+            check_id='custom_aws_12345',
+            bc_check_id='custom_aws_12345',
+            check_result={'result': CheckResult.FAILED, 'evaluated_keys': ['map_public_ip_on_launch']},
+            check_name='Cloned-Ensure VPC subnets do not assign public IP by default',
+            check_class='checkov.terraform.checks.resource.aws.SubnetPublicIP',
+            code_block=[(1, 'resource "aws_subnet" "s" {\n'), (2, '  map_public_ip_on_launch = true\n'), (3, '}\n')],
+            evaluations=None,
+            file_abs_path=file,
+            file_line_range=[2, 3],
+            file_path='/main.tf',
+            resource='aws_subnet.s'
+        ))
+
+        fixes_integration.post_runner(report)
+
+        self.assertTrue(all(r.fixed_definition is not None for r in report.failed_checks))
+
+    def setUp(self) -> None:
+        global _old_check_metadata
+        _old_check_metadata = metadata_integration.check_metadata
+
+    def tearDown(self) -> None:
+        metadata_integration.check_metadata = _old_check_metadata
```

Comment:
```suggestion
    def setUp(self) -> None:
        self._old_check_metadata = metadata_integration.check_metadata

    def tearDown(self) -> None:
        metadata_integration.check_metadata = self._old_check_metadata
```
this should do the trick without needing to use global 🙂